src/st/cen/hd.c

```diff
@@ -0,0 +1,278 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include "common.h"
+
+#if defined(VERSION_HD)
+
+INCLUDE_ASM("st/cen/nonmatchings/hd", EntityUnkId12);
```

Comment:
The previous time we added HD to a stage it looked like this https://github.com/Xeeynamo/sotn-decomp/pull/1421/files

Xeeynamo would chime in on whether he wants this done on a branch or incrementally on master